routers/api/v1/misc/settings.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package misc
+
+import (
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/setting"
+)
+
+// SettingGetsAllowedReactions return allowed reactions
+func SettingGetsAllowedReactions(ctx *context.APIContext) {
+	// swagger:operation GET /settings/allowed_reactions miscellaneous getAllowedReactions
+	// ---
+	// summary: Returns string array of allowed reactions
```

Comment:
Not sure if needed to clarify since it is covered in the config docs, but reactions can be either an alias like 'rocket' or the actual emoji '🚀' which might not be obvious

Reply:
This schould not have any issue ...